File: solutions/javascript/reverse-string/1/reverse-string.js
//
// This is only a SKELETON file for the 'Reverse String' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const reverseString = (str) => {
  let index_start = 0
  let nova_str = ""
  for (let index = str.length -1; index >= 0; index--) {
    nova_str += str[index]
    index_start++
  }
  return nova_str
};

File: solutions/cpp/reverse-string/1/reverse_string.cpp
#include "reverse_string.h"
using namespace std;
namespace reverse_string {
    string reverse_string (string s) {
        string nova_str = "";
        for (int i = s.length()-1; i>=0; i--) {
            nova_str = nova_str + s[i];
        }
        return nova_str;
    }
}  // namespace reverse_string
